VERSION

```diff
@@ -1 +1 @@
-2020.09.0
+2024.09.0
```

Comment:
[CHANGELOG](https://github.com/aws/amazon-ecs-cni-plugins/blob/master/CHANGELOG.md) seems to have not been updated with previous few commits. However since this change seems to be updating version, we can update that too

Reply:
Updated CHANGELOG in [5a24ed4](https://github.com/aws/amazon-ecs-cni-plugins/pull/113/commits/5a24ed4da9a194a83b6131c894e276cb2256cafe).